common/src/components/accordion/Accordion.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from "react";
+import { ChevronDownIcon, ChevronRightIcon } from "@heroicons/react/solid";
+
+export interface AccordionProps {
+  title: string;
+  subtitle?: string;
+  Icon?: React.ComponentType<React.SVGProps<SVGSVGElement>>;
+  context?: string;
+  simple?: boolean;
+  defaultOpen?: boolean;
+}
+
+interface InternalAccordionProps extends AccordionProps {
+  open?: boolean;
+}
+
+export const Accordion: React.FC<InternalAccordionProps> = ({
+  title,
+  subtitle,
+  simple,
+  Icon,
+  children,
+  defaultOpen,
+}) => {
+  const contentRef = React.useRef(null);
```

Comment:
Does this ref have a use, or is it left over from previous versions?

Reply:
removed it